fvm/src/kernel/error.rs

```diff
@@ -60,8 +60,8 @@ impl ExecutionError {
     pub fn is_recoverable(&self) -> bool {
         use ExecutionError::*;
         match self {
-            OutOfGas | Fatal(_) => false,
             Syscall(_) => true,
+            OutOfGas | Fatal(_) => false,
```

Comment:
revert (probably my fault)

Reply:
I think we can just delete this method as its not used anymore